CMakeLists.txt

```diff
@@ -20,21 +20,22 @@ set(CMAKE_C_FLAGS_DEBUG "${CMAKE_C_FLAGS_DEBUG} -O0")
 
 # add the executable
 
-## cfe + aorc + ftm
-#add_executable(cfeft ./src/main_pass_forcings.cxx ./src/cfe.c ./src/bmi_cfe.c ./forcing_code/src/aorc.c ./forcing_code/src/bmi_aorc.c)
+## cfe + aorc + pet + ftm
+add_executable(cfeft ./src/main_cfe_aorc_pet_ftm.cxx ./cfe/src/cfe.c ./cfe/src/bmi_cfe.c ./cfe/forcing_code/src/aorc.c ./cfe/forcing_code/src/bmi_aorc.c ./cfe/forcing_code/src/pet.c ./cfe/forcing_code/src/bmi_pet.c)
```

Comment:
Is the path to the first one correct (i.e. ./src/main_cfe_pet_ftm.cxx vs ./**cfe**/src/main_cfe_aorc_pet/ftm.cxx)?

Reply:
yes, the pass is correct.

Reply:
*path